src/shogun/base/class_list.cpp.templ

```diff
@@ -70,3 +70,17 @@ std::set<std::string> shogun::available_objects()
 	}
 	return result;
 }
+
+std::string shogun::find_correct_name(const std::string& name)
+{
+    using pair = std::pair<int32_t, std::string>;
+    std::priority_queue<pair, std::vector<pair>, std::greater<pair>> names;
+    LevenshteinDistance levenshtein;
```

Comment:
on stack? Don't we do shogun objects on heap usually?

Reply:
probably should, I think we should be consistent even if here it isn't needed